### DISCUSSION TITLE
UI improvement: Add the save&reboot button in the header

### DISCUSSION THREAD

**Discussion by neuhausj** [ExpressLRS/ExpressLRS#2122] [Ideas]
Hello,

The current place of the Save&Reboot button could be improved. 
Currently it looks like it is only valid for the Runtime options. 
Not for Wifi / Model match config for example.
Moreover, a page switch is required to save the settings.

Maybe adding it directly to the banner or header to make it visible from all tabs could be a nice addition.

Let me know what you think,

![image](https://user-images.githubusercontent.com/182152/224947164-a3e7a4e4-7930-41a7-951c-99c4fe3b9bd8.png)

**Comment by pkendall64** [comment]
Save and reboot is not required for wifi settings or model match.

**Reply by neuhausj** [reply]
I agree, but my point is how to simply reboot the Rx/Tx ?
This should be available from any tab to be able to switch back to "live" mode.
